### PR TITLE
[HPRO-791] Migrate data clean up cron jobs

### PR DESCRIPTION
**Pull request by shyamnaru**
| Q                   | A
| ------------------- | --------------
| Target Release?     | Next available 
| Bug fix?            | ❌ 
| New feature?        | ❌ 
| Database migration? | ❌ 
| New configuration?  | ❌ 
| Composer updates?   | ❌ 
| NPM updates?        | ❌ 
| Jira Ticket(s)      | HPRO-791

### CODE REVIEW

**Review by stephenyeargin — APPROVED** [review]

File: symfony/src/Service/SessionService.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?php
```
Comment (jt2k) [inline symfony/src/Service/SessionService.php:1]: Can the `src/Pmi/Service/SessionService.php` class be removed now?

File: symfony/src/Service/SessionService.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    private $params;
+
+    public function __construct(ParameterBagInterface $params)
+    {
+        $this->params = $params;
+    }
+
+    public function deleteKeys()
+    {
+        $sessionTimeout = 30 * 60;
```
Comment (jt2k) [inline symfony/src/Service/SessionService.php:19]: Can we get this from the environment service since it can change based on environment?

**Review by jt2k — CHANGES_REQUESTED** [review]

File: symfony/src/Service/EnvironmentService.php [hunk 3]
```diff
@@ -171,4 +171,10 @@ protected function loadConfiguration($override = [])
             $this->configuration[$key] = $val;
         }
     }
+
+    public function getSessionTimeout()
+    {
+        return $this->isLocal() ? 3600 * 24 : 30 * 60;
```
Comment (jt2k) [inline symfony/src/Service/EnvironmentService.php:177]: Isn't this already defined in the constructor on line 49?

Comment (shyamnaru) [reply symfony/src/Service/EnvironmentService.php:177]: Yeah you are right, I'll fix it. Thanks!

**Review by jt2k — APPROVED** [review]

**Pull request merged**